### PR TITLE
Bump chart version and NuoDB version

### PR DESCRIPTION
**Pull request by adriansuarez**

### CODE REVIEW

**Review by kontaras — APPROVED** [review]

File: incubator/demo-ycsb/Chart.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ maintainers:
   email: support@nuodb.com
 engine: gotpl
 icon: https://raw.githubusercontent.com/nuodb/nuodb-helm-charts/master/images/nuodb.svg
-appVersion: 5.0.4
+appVersion: "5.1"
```
Comment (sivanov-nuodb) [inline incubator/demo-ycsb/Chart.yaml:15]: The pinned NuoDB image tag seems to be 5.0.1.

**Review by sivanov-nuodb — APPROVED** [review]
Looks good.

File: incubator/demo-ycsb/Chart.yaml (hunk 1 above)
Comment (adriansuarez) [reply incubator/demo-ycsb/Chart.yaml:15]: Good catch.

**Comment by adriansuarez** [comment]
I have to close this PR because GitHub refuses to synchronize it with my branch.

**Pull request closed without merging**